tldw_Server_API/tests/Services/test_main_shutdown_job_pollers.py

```
@@ -0,0 +1,593 @@
+    monkeypatch.setattr(main_module, "_stop_registered_job_pollers", _fake_stop_pollers)
+
+    await main_module._quiesce_owned_job_pollers_for_shutdown(
+        app,
+        [],
+        wait_for_leases_sec=5,
+        count_active_processing=lambda: next(counts),
+    )
+
+    assert observed_sleeps == [0.5, 0.5]
+    assert stop_calls == ["stop"]
+    segments = getattr(app.state, "_tldw_shutdown_timing_segments")
```

Ruff (0.15.10)

[warning] Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)